trailblazer/dto/analysis_response.py

```diff
@@ -27,24 +27,27 @@ class Job(BaseModel):
 class AnalysisResponse(BaseModel):
```

Comment:
I put the fields in alphabetical order but I urge anyone interested to double-check if I messed up.